Codeinwp/templates-patterns-collection
# Have TPC Neve independent

### Summary


- [x] Remove the Notice that Neve is required from Plugins
- [x] I will remove the pop-up when Neve is not present on Starter Sites and add Neve as part of the required Install.
- [x] For Page Templates I will let them import the template without requiring them to have Neve.
- [x] I will ensure that if the Neve Pro license is available, the Template Cloud is available even if the Neve theme is not being used. My Library should be available on other themes if the License is active.
- [x] Show Upsell modal for Page Templates if Neve is not present
- [x] Change Upsell modal for Cloud Library if Neve is not present
- [x] Update Import modal, add a separate section for Theme to make it more obvious that the theme will be replaced.    

### Screenshots
Import Modal when Neve is not installed:
![image](https://user-images.githubusercontent.com/23024731/181007108-8fd74f46-7ac5-4b9b-84f2-ff39e175b97c.png)


Upsell modal when Neve is not present for Page Templates or Cloud Library:
![image](https://user-images.githubusercontent.com/23024731/181007248-f78d97dd-dfad-4b90-92fd-778194d53c64.png)


Upsell modal if Neve is present for Cloud Library:
![image](https://user-images.githubusercontent.com/23024731/181007368-63ad458f-4ef1-453a-8f25-d99b5c8d4b7b.png)



### Test instructions


1. Check that TPC works as before with Neve and Neve Pro installed
2. Check that without Neve, the Starter Sites will install Neve and present that on Import
3. Check that without Neve, the Page Templates are not accessible, an Upsell is presented instead
4. Check that without Neve, the Cloud Library has a modified Upsell
5. Check that if you had Neve and Neve Pro active, for Neve Pro use this PR: https://github.com/Codeinwp/neve-pro-addon/pull/2087, see that Cloud Library is available if changing themes.


Closes #154.
